### PR TITLE
fix cm4s specific configuration get applied on CM3[REVPI-2305]

### PR DESCRIPTION
**Pull request by zhan-kunbus**

### CODE REVIEW

File: templates/config.txt [hunk 1]
```diff
@@ -56,8 +56,10 @@ hdmi_drive=2
 # Enable audio (loads snd_bcm2835)
 dtparam=audio=on
```
Comment (iluminat23) [inline templates/config.txt:58]: This is from the current upstream config.txt:
```suggestion

# Enable DRM VC4 V3D driver
#dtoverlay=vc4-kms-v3d
#max_framebuffers=2

```

File: templates/config.txt [hunk 2]
```diff
@@ -56,8 +56,10 @@ hdmi_drive=2
...
 
+[pi4]
 [cm4s]
 dtoverlay=dwc2,dr_mode=host
 #dtoverlay=vc4-kms-v3d
 #max_framebuffers=2
```
Comment (iluminat23) [inline templates/config.txt:63]:
```suggestion
```

**Review by iluminat23 — CHANGES_REQUESTED** [review]
The functional changes seem fine. I've tested it on an Connect+ and on a Core with CM4S. Just move the lines about the framebuffer up.

**Comment by iluminat23** [comment]
https://www.raspberrypi.com/documentation/computers/config_txt.html#otg_mode-raspberry-pi-4-only

**Comment by zhan-kunbus** [comment]
> https://www.raspberrypi.com/documentation/computers/config_txt.html#otg_mode-raspberry-pi-4-only

And what's the point?   It seems it is only rpi4 relevant, and not for us at moment (CM4S)?

**Comment by iluminat23** [comment]
> And what's the point? It seems it is only rpi4 relevant, and not for us at moment (CM4S)?

It is a different USB controller which is included in the BCM2711. It is an xhci USB 2.0 controller. I do some tests now to check what is different. Give me a few minutes.

**Comment by iluminat23** [comment]
Move the commit 1e06bc5575b192830bb812e7c293cc1dbcdaf5fa to the 2nd commit. Also add a signed-off and remove the co-authored github@iluminat23.org. I know github adds these, but it doesn't make any sense here. Or at least add my kunbus mail address.

**Comment by iluminat23** [comment]
The `otg_mode=1` uses a different driver, but brakes our udev rules. So we need to stay on the old driver for now.

**Review by iluminat23 — CHANGES_REQUESTED** [review]
Move the commit https://github.com/RevolutionPi/imagebakery/commit/1e06bc5575b192830bb812e7c293cc1dbcdaf5fa to the 2nd commit. Also add a signed-off and remove the co-authored github@iluminat23.org. I know github adds these, but it doesn't make any sense here. Or at least add my kunbus mail address.

**Pull request merged**